src/katsdpcalproc/pointing.py

```diff
@@ -329,6 +317,11 @@ def calc_pointing_offsets(session, beams, target, middle_time,
         find the mean location of a moving target (and reference for weather)
     temperature, pressure, humidity : float
         Atmospheric conditions at middle time, used for refraction correction
+    ants: A list containing <class: katpoint.Antenna> objects for each antenna
```

Comment:
The order in the `Parameters` section in the docstring should match the order in the actual argument list.

Also, have a look at the way the other entries are formatted. It follows:
```
name: type
    description on a new line
```
The "type" uses Sphinx notation for classes like ":class:`Class`". Therefore you should have something like:
```
ants: list of :class:`katpoint.Antenna`
    Antenna objects for fitted beams
```